### DISCUSSION TITLE
Need a little help suppressing output for the first second.

### DISCUSSION THREAD

**Discussion by Karma-Tron** [official-stockfish/Stockfish#4011] [Q&A] [answered]
I am trying to suppress output for the first second by adding  `|| elapsed < 1000` at line 1862 in search.cpp:

`      if ((depth == 1 && !updated && i > 0) || elapsed < 1000)
          continue;
`

This works, however I still get linefeeds for the suppressed output and cannot figure out where those are coming from.
Any insight is greatly appreciated.

**Comment by vondele** [comment, marked as answer]
you are still passing an empty string to the various places this is used. This is the wrong place to try and fix this. Probably add something to line 424 (and possibly other places that uses this function)?

**Comment by Karma-Tron** [comment]
Thank you, several additions and it works like a charm !
